src/app/tasks/tasks.component.html

```
@@ -25,14 +26,12 @@
           <span matListItemLine><ng-container i18n>Completed:</ng-container> {{task.completedTime | date}} {{task.completedTime | date: 'shortTime'}}</span>
         }
-        @if (task.assignee) {
-<img matListItemAvatar (click)="openMemberDialog(task.assignee); $event.stopPropagation()" matTooltip="{{task.assignee | assigneeName}}" [src]="
-      task.assignee.attachmentDoc ?
-        imgUrlPrefix + '/attachments/' + task.assignee.attachmentDoc._id + '/' + (task.assignee.attachmentDoc._attachments.img ? 'img' : 'img_') :
-        task.assignee.avatar ?
-        imgUrlPrefix + task.assignee.avatar :
-        'assets/image.png'
-    ">
+        @if (taskView.assignee; as assignee) {
+          <img matListItemAvatar
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)